tests/integration_tests/validation/conformance_suite_config.py

```diff
@@ -27,15 +28,21 @@ class ConformanceSuiteConfig:
     extract_path: str | None = None
     membership_url: str | None = None
     nested_filepath: str = ''
+    packages: list[str] = field(default_factory=list)
     plugins: frozenset[str] = frozenset()
     public_download_url: str | None = None
     shards: int = 1
     strict_testcase_index: bool = True
     url_replace: str | None = None
     network_or_cache_required: bool = True
     required_locale_by_ids: dict[str, re.Pattern[str]] = field(default_factory=dict)
+    test_case_result_options: Literal['match-all', 'match-any'] = 'match-all'
 
     def __post_init__(self) -> None:
+        packages = [f'https://arelle-public.s3.amazonaws.com/ci/taxonomy_packages/{package}' for package in self.packages]
+        # self.packages = packages
```

Comment:
Not sure if you left this here to describe the intent of the next line, but thought I'd point it out. I don't care either way.